charts/weblate/templates/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.global.translationManagementSystemEnabled }}
```

Comment:
Add weblate to the main Chart.yaml file in the root to disable yaml so you don't have to adapt all files.
```
dependencies:
  - name: online-counseling-weblate
    version: 0.5.3
    condition: weblate.enabled
```

Reply:
And can we add weblate as git submodule? Because updates will be simpler.

Reply:
it did not work for me out of the box, I needed to adopt the contents of some charts to get it work